.ci_support/linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11numpy1.26python3.12.____cpython.yaml

```diff
@@ -0,0 +1,39 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '11'
+cdt_name:
+- cos7
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge openmm_rc
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- '11.8'
```

Comment:
Interestingly [this CI build passes]( linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11numpy1.26python3.12.____cpython.yaml ) though the values are exactly the same as the previous variant (both as would be expected for CUDA 11.8)